pkg/application/app.go

```diff
@@ -68,6 +68,10 @@ func (app *Avalanche) GetKeyDir() string {
 	return filepath.Join(app.baseDir, constants.KeyDir)
 }
 
+func (app *Avalanche) GetPluginDir() string {
```

Comment:
GetPluginDir may be misleaving as avalanchego is not going to use this directory

Reply:
renamed